src/components/FAQItem.svelte

```diff
@@ -14,7 +14,7 @@
 
 <button
   id="faq-question"
-  class="label text-md mt-2 flex w-full items-center justify-between border-t border-divider p-4 text-left"
+  class="label border-divider mt-2 flex w-full items-center justify-between border-t p-4 text-left text-md"
```

Comment:
Additionally, it doesn't seem to match the style guide. Either import the one from the SG, or abide by the styles here: https://www.figma.com/design/EjFt1XdacOogDZoQ7RetrI/Frequency-Style-Guidelines?node-id=1072-1096&node-type=frame&t=M9rvMiiDpjs2zIh5-0